lib/tokens/token.js

```diff
@@ -59,6 +59,18 @@ module.exports = function (log, random, P, hkdf, Bundle, error) {
   // This uses randomly-generated seed data to derive the keys.
   //
   Token.createNewToken = function(TokenType, details) {
+    if (details.createdAt) {
+      // We don't expect this to be set outside the tests so log an error
+      // if we do encounter it, to help debug what's going on.
+      const err = new Error('Unexpected createdAt data')
+      log.error({
+        op: 'token.constructor',
+        TokenType: TokenType.name,
```

Comment:
naming nit: Should this be `tokenType`? Couldn't find any references of logs using capitalize letter to start.